legend-engine-pure/legend-engine-pure-code/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/graphFetch/graphFetch_routing.pure

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import meta::relational::mapping::*;
```

Comment:
How is this import used?  Can it be removed?

Seems that this is breaking encapsulation - core libraries depending on the implementation of relational.

Reply:
yes, this can be removed.